### ISSUE TITLE
Latest Version of testng 7.7.1 is compatible with java11.Doesnot supports Java8. Release a version of testng which is compatible with java8 and has no vulnerabilities like testng 7.7.1

### ISSUE THREAD

**Issue by immi543** [cbeust/testng#2901]
### TestNG Version
> Note: only the latest version is supported

Latest version of testng 7.7.1 doesnot supports Java8. Earlier version which supports java8 have vulnerabilities.

### Expected behavior

Please release a version of testng which is compatible with Java8 as well.

### Is the issue reproducible on runner?

- [ ] Shell
- [x] Maven
- [ ] Gradle
- [ ] Ant
- [x] Eclipse
- [ ] IntelliJ
- [ ] NetBeans

### Test case sample
> Please, share the test case (as small as possible) which shows the issue

Code doesnot compile with latest testng jar if user is on java8. Earlier version of testng which supports java8 are blocked by jfrog because they have vulnerabilities.

### Contribution guidelines

Incase you plan to raise a pull request to fix this issue, please make sure you refer our [Contributing](.github/CONTRIBUTING.md) section for detailed set of steps.

**Comment by krmahadevan** [comment]
@immi543 - We strongly urge you to upgrade to JDK11. Its not going to be possible to keep patching earlier released version of TestNG with security vulnerability fixes. Today there is one, there's going to be more tomorrow and its not going to be possible for us to keep doing this.

On the other side, unless and until you are using the `-testjar` option of TestNG wherein you are building a fat jar of your tests and then passing that jar as parameter to TestNG for execution, you are NOT exposed to any security vulnerability.

If you are still looking for 7.5 support with JDK8, then I would suggest that you follow this PR https://github.com/cbeust/testng/pull/2899

**Comment by krmahadevan** [comment]
Duplicate of https://github.com/cbeust/testng/issues/2895
